client/src/components/Post/Post.tsx

```diff
@@ -132,6 +255,21 @@ class Post extends Component<Props, State> {
             </div>
         )
     }
+
+    private getOnClick(props: any & { submitForm: () => Promise<any> }) {
+        return () => {
+            this.handlePostComment(props.values.comment);
+            var commentArray= {id: 0,
+                author: 'Jhon Mark',
+                body: props.values.comment,
+                date: '20/20/20 18:00 Hs'}
+            var comments= this.state.comments;
+            comments.push(commentArray);
```

Comment:
This should not be hardcoded. We should push the new comment to the state of comments. Also please use let instead of var.